packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -151,31 +153,16 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const localeStrings = useLocale().strings;
 
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
-  const [textOnlyChat, setTextOnlyChat] = useState(false);
+  const textOnlyChat = !!useSelector(getTextOnlyChat);
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
-  const [isACSChat, setIsACSChat] = useState(false);
+  const createdBy = useSelector(getCreatedBy);
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  const isACSChat = !createdBy || !isMicrosoftTeamsUserIdentifier(createdBy);
```

Comment:
```suggestion
  const isACSChat = !!createdBy && !isMicrosoftTeamsUserIdentifier(createdBy);
```

Reply:
Just to confirm, you want to default to this being a teams chat if createdBy doesn't exist?

Reply:
`createdBy` should always be provided, the only case when it's underfined is when we have thread but haven't requested and in this case we don't rel]ally know if it's an ACS chat or Teams meeting

Reply:
Maybe we can change the logic a little bit to have
`const isACSChat = createdBy && !isMicrosoftTeamsUserIdentifier(createdBy);`

and there should be 1 place where `isACSChat`is used and update it to 
`const onPasteCallback = !!isACSChat || textOnlyChat ? removeImageTags : undefined;`

Reply:
@JamesBurnside please let me know if you prefer me to submit it as a PR (as it's additional changes)

Reply:
np! updated